orchester_cms_integration/templates/orchester/edit.html

```diff
@@ -39,6 +39,56 @@ <h2>Services</h2>
     var unregisterUrl = "{% url 'orchester:unregister' member.id %}";
     var registerUrl = "{% url 'orchester:register' member.id %}";
 
+    function renderNormalService(data) {
+      return data.status
```

Comment:
Instead of duplicating the template in the ternary, can't we just do :
`let actionInput = data.status ? '<input type="submit" class="button--primary" value="Unregister"/>' : '<input type="submit" class="button--primary" value="Register"/>';`
then just return
```
[
'<form class="orchester__edit__buttonForm" action="' + unregisterUrl + '" method="post">',
'<input type="hidden" name="connector_name" value="' + data.name + '" />',
"{% csrf_token %}",
actionInput,
'</form',
]
.join('')
```

Reply:
samething with the `form`, since we just change if it's the `unregisterUrl` or `registerUrl`